api.go

```diff
@@ -73,7 +68,7 @@ func (conn *Connection) rqliteApiCall(apiOp apiOperation, method string, request
 		}
 
 		// Read response body even if not a successful answer to return a descriptive error message
-		responseBody, err := ioutil.ReadAll(response.Body)
+		responseBody, err := io.ReadAll(response.Body)
```

Comment:
why is this change needed?

Reply:
`ioutil` is deprecated per 1.16 https://pkg.go.dev/io/ioutil#ReadAll

Reply:
Ah, good to know!